wolfcrypt/src/port/Espressif/esp_sdk_time_lib.c

```diff
@@ -151,7 +151,7 @@ int set_fixed_default_time(void)
     struct tm timeinfo = {
```

Comment:
Might consider a compile-time solution like this -> https://github.com/wolfSSL/wolfssl/pull/7993/files#diff-f150aa374cc550805c2df21b76802b721e709e34d36209c60cc9ee82da22b8b4R24

Reply:
Oh, that's nice! Added. Thanks for that tip.

Reply:
p.s. the `__DATE__` is not always defined, so there are still hard coded values. 

See comment on Espressif "reproducible builds".